cdc-backfill-client/src/test/java/com/datastax/oss/cdc/backfill/PulsarImporterTest.java

```diff
@@ -99,7 +99,6 @@ public void testImportPartitionKeyOnly() throws Exception {
         // given
         Config connectorConfig = createConnectorConfigs("sample-001.csv");
         connector.configure(connectorConfig, true, true);
-        connector.init();
```

Comment:
why are we removing this ?

Reply:
in DSBulk, the init() is responsible for calculating the directory/file structure of the exported table so the init should happen lazily. I moved it to when the actual import is called - will add unit tests for it. Alternatively I can keep it here and just inject a factory class into the importer.